### ISSUE TITLE
[1.12.2] Strange fog behaviour makes game unplayable

### ISSUE THREAD

**Issue by Insane96** [TheCyberBrick/The-Betweenlands#433]
Today I've rejoined my world and this happened.
![2017-12-11_20 49 15](https://user-images.githubusercontent.com/7185104/33851239-ac90d6b8-deb6-11e7-9b9c-4749c7b8b209.png)
Then I managed to get back to the overworld
![2017-12-11_20 52 52](https://user-images.githubusercontent.com/7185104/33851263-b82abd0e-deb6-11e7-8497-76fe1a6ced5b.png)
Nothing. I've tried removing other mods (Quark and Optifine) and still had the issue, the only mod that I kept was JEI.
Then got back to the Betweenlands and seems like that disabling clouds would do the job, sadly as soon as I got back to the Overworld, the problem persisted, and changing clouds settings did nothing.

Rebooted minecraft and opened the world again, clouds were to Fancy. Overworld was normal, as soon as I got to the Betweenlands the problem presented again. Disabled Clouds and the overworld wasn't workin again. Got back to the betweenlands, set Clouds to Fancy, it worked! and as soon as I got back to the overworld it kept working, back to the betweenlands and still works.

I'm writing this here for anyone that has the same problem and to report this strange bug.

TLDR: If you're having some strange gray thing on blocks (I think is actually fog rendered in the opposite way) disable clouds ("Clouds: OFF") in video settings seems to fix it for some reasons. Still didn't check with Optifine

**Comment by TheCyberBrick** [comment]
Hm interesting, I ran into this bug myself a while ago but then never again. This kind of thing usually happens when the fog start/end is a negative number, will have a look at the fog code

**Comment by TheCyberBrick** [comment]
This should have been fixed a while ago IIRC, if it reappears feel free to open a new issue
